tests/subsys/fs/littlefs/src/test_lfs_basic.c

```diff
@@ -62,10 +62,23 @@ static int clean_statvfs(const struct fs_mount_t *mp)
 		 stat.f_bsize, stat.f_frsize, stat.f_blocks, stat.f_bfree);
 	zassert_equal(stat.f_bsize, 16,
 		      "bsize fail");
+#ifdef CONFIG_SCORPIO_FLASH
```

Comment:
Looks good to me, we can always make the two numbers into config vars if needed, for now the comment explains things well :)